apps/web/src/pages/integrations/useProviders.ts

```diff
@@ -23,36 +34,38 @@ function fcmFallback(integration: IntegrationEntity | undefined, clonedCredentia
 }
 
 function initializeProviders(integrations: IntegrationEntity[]): IIntegratedProvider[] {
-  return providers.map((providerItem) => {
-    const integration = integrations.find((integrationItem) => integrationItem.providerId === providerItem.id);
+  return providers
```

Comment:
The filtering of novu providers could also be done here,  but not significant

Reply:
yeah, better to move to here, will do